core/README.md

```diff
@@ -2,23 +2,77 @@
+endpoint to be able to search across items in its collections. Note though that none of those links are required, other servers may provide
+different conformance classes and a different set of links. 
+
+The only requirements of the STAC API core class are to provide a valid STAC Catalog that includes a valid `conformsTo` JSON object
+in it. Any API implementing that is considered a valid STAC API, and clients can inspect the document to figure out what other
+capabilities are on offer and how to reach them.
+
+The root endpoint (`/`) is most useful when it presents a complete `Catalog` representation of all the data contained in the API, such 
+that all `Collection` and `Item` objects can be navigated to by transitively traversing links from this root. This spec does not require any 
+API endpoints from OAFeat or STAC API to be implemented, so the following links may not exist if the endpoint has not been implemented.
+
+## Recommended Link Relations at `/`
+
+When implementing the STAC API Core conformance class, it it recommended to implement these Link relations.
+
+| **`rel`** | **href to**                                | **From**           | **Description**                                                  |
+|-----------|--------------------------------------------|--------------------|------------------------------------------------------------------|
+| `root`    | The root URI                               | STAC Core          | Reference to self URI |
+| `self`    | The root URI                               | OAFeat             | Reference to self URI  |
+| `service-desc` | The OpenAPI service description       | OAFeat OpenAPI   | Uses the `application/vnd.oai.openapi+json;version=3.0` media type to refer to the OpenAPI 3.0 document that defines the service's API |
+| `service-doc`  | An HTML service description           | OAFeat OpenAPI   | Uses the `text/html` media type to refer to a human-consumable description of the service |
+| `child`   | The child STAC Catalogs & Collections      | STAC Core          | Provides curated paths to get to STAC Collection and Item objects      |
+
+It is also valid to have `item` links from the landing page, but most STAC API services are used to 
+serve up a large number of features, so they typically
+use several layers of intermediate `child` links before getting to Item objects.
+
+## Example Landing Page for STAC API - Core
+
+This document is what would be expected from an api that only implements STAC API - Core. In practice, 
+most APIs will also implement other conformance classes, and those will be reflected in the `links` and `conformsTo` fields.
 
 ```json
 {
     "stac_version": "1.0.0-beta.2",
     "id": "example-stac",
     "title": "A simple STAC API Example",
     "description": "This Catalog aims to demonstrate the a simple landing page",
+    "conformsTo" : [
```

Comment:
I moved conformsTo to the top of all the examples, since this is pretty critical to how most of the clients and validators are going to interact with the API